src/index.tsx

```diff
@@ -314,6 +318,10 @@ class RichMarkdownEditor extends React.PureComponent<Props, State> {
             dictionary,
             onShowToast: this.props.onShowToast,
           }),
+          new Emoji({
```

Comment:
You will need add emoji to disableExtensions in props.

Reply:
thx, i will~